app/models/post.rb

```diff
@@ -3,6 +3,7 @@ class Post < ApplicationRecord
   has_many :likes
   has_many :comments
   after_create :increment_user_posts_counter
+  after_save :update_post_counter
```

Comment:
- [OPTIONAL] Kindly consider adding everything in the suggestion of the previous reviewer and remove the `increment_user_posts_counter` method as you will be implementing the `:update_post_counter` method in its place or just ignore his suggestion and remove line 6 because if you have not written the `:update_post_counter` method then calling that method is illogical.